app/views/external_users/claims/expenses/_expense_fields.html.haml

```diff
@@ -1,134 +1,114 @@
-            = f.select :expense_type_id, expense_types_collection.map { |et| [et.name, et.id, { data: et.data_attributes }] }, { include_blank: t('.select_option') }, { class: 'form-control form-control-3-4' }
-            = validation_error_message(@error_presenter, "expense_#{@expense_count}_expense_type")
-
-        .govuk-grid-column-full
-          -# SELECT: Travel reasons
-          - error_classes = error_class?(@error_presenter, "expense_#{@expense_count}_reason_id")
-          - error_classes += ' form-group-error' if error_classes.present?
-          %a{ id: "expense_#{@expense_count}_reason_id" }
-          .form-group.fx-travel-reason{ style: 'display: none;', class: error_classes }
-            = f.label :reason_id, class: 'form-label-bold' do
-              = t('.travel_reason_html')
-              = validation_error_message(@error_presenter, "expense_#{@expense_count}_reason_id")
-
-            - reasons_collection = present_collection(reasonset_for_expense_type(f.object.expense_type).values, TravelReasonPresenter)
-            = f.select :reason_id, reasons_collection.map { |et| [et.reason, et.id, { data: et.data_attributes }] }, { include_blank: t('.select_option') }, { class: 'form-control form-control-3-4' }
-
-          -# INPUT: Other travel reason input
-          .form-group.fx-travel-reason-other{ style: f.object.reason_id.eql?(5) ? 'display: block;' : 'display: none;' }
-            - present(f.object) do |expense|
-              = f.adp_text_field :reason_text, label: t('.reason_text_html'), input_classes: '', errors: @error_presenter
-
       .govuk-grid-row
         .govuk-grid-column-full
-          -# INPUT: Location
-          .fx-travel-location{ style: 'display: none;' }
-
+          - expense_types_collection = present_collection(ExpenseType.for_claim_type(@claim))
+          = f.govuk_collection_select :expense_type_id,
+            expense_types_collection.map { |et| [et.name, et.id, { data: et.data_attributes }] },
+            :second,
+            :first,
+            form_group: { classes: 'fx-travel-expense-type' },
+            label: { text: t('.type_of_expense_html') },
+            options: { include_blank: t('.select_option') }
+
+          - reasons_collection = present_collection(reasonset_for_expense_type(f.object.expense_type).values, TravelReasonPresenter)
+          = f.govuk_collection_select :reason_id,
+            reasons_collection.map { |et| [et.reason, et.id, { data: et.data_attributes }] },
+            :second,
+            :first,
+            form_group: { classes: 'fx-travel-reason', style: 'display: none' },
+            label: { text: t('.travel_reason_html') },
+            options: { include_blank: t('.select_option') }
+
+          - present(f.object) do |expense|
+            = f.govuk_text_field :reason_text,
+              form_group: { classes: 'fx-travel-reason-other', style: f.object.reason_id.eql?(5) ? 'display: block' : 'display: none' },
```

Comment:
`f.object.reason_id.eql?(5) ? 'display: block' : 'display: none'` seems to be redundant code.
This ternary would only work on the server side, but this entire step uses client side script to work.

Note: Please create a ticket to rewrite this form step to follow GOV's progressive enhancement strategy